### PR TITLE
UX: Icons: Remove usage of chevron fa- icons

### PR DESCRIPTION
**Pull request by darkwing**
## Explanation

We present use `fa-chevron{direction}` icons throughout the app.  This PR replaces usages of those with our new icons.

### Sender To Recipient
ChevronSenderToRecipient

### Popover
ChevronPopover

### Dropdown 
ChevronDropdowns

### Activity Log Icons
ChevronActivityLogIcons

### Confirmation Back button
ChevronConfirmationBack

### Send flow
SendFlow

### Breadcrumbs
ChevronCrumb

### NFTs
NFTChevrons

### Asset List Items
AssetListItems

### Network Picker
ChevronNetworkPicker

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/tab-bar/tab-bar.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const TabBar = (props) => {
             <div className="tab-bar__tab__content__icon">{icon}</div>
             <div className="tab-bar__tab__content__title">{content}</div>
           </div>
-          <IconCaretRight className="tab-bar__tab__caret" />
+          <Icon name={ICON_NAMES.ARROW_RIGHT} className="tab-bar__tab__caret" />
         </button>
       ))}
     </div>
```
Comment (georgewrmarshall) [inline ui/components/app/tab-bar/tab-bar.js:27]: After

Screenshot 2023-02-08 at 6 12 20 PM

File: ui/components/app/network-display/network-display.js [hunk 2]
```diff
@@ -66,11 +66,7 @@ export default function NetworkDisplay({
...
       }
-      rightIcon={
-        onClick ? (
-          <IconCaretDown size={16} className="network-display__icon" />
-        ) : null
-      }
+      rightIcon={onClick ? <Icon name={ICON_NAMES.ARROW_DOWN} /> : null}
       label={
         networkType === NETWORK_TYPES.RPC
           ? networkNickname ?? t('privateNetwork')
```
Comment (georgewrmarshall) [inline ui/components/app/network-display/network-display.js:72]: Just adding some sizing here

Screenshot 2023-02-08 at 6 10 23 PM
Screenshot 2023-02-08 at 6 10 49 PM

File: ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js [hunk 3]
```diff
@@ -31,24 +33,16 @@ export default class TransactionActivityLogIcon extends PureComponent {
...
-              'fa',
-              'transaction-activity-log-icon__icon',
-              iconClassName,
-            )}
-          />
-        ) : null}
+        {icon ? <Icon name={icon} color={Color.iconDefault} /> : null}
       </div>
     );
   }
```
Comment (georgewrmarshall) [inline ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js:48]: After


https://user-images.githubusercontent.com/8112138/217699598-778f4ec7-534d-4f09-9b5c-f05b64b1176a.mov

File: ui/components/ui/dropdown/dropdown.js [hunk 4]
```diff
@@ -41,7 +41,10 @@ const Dropdown = ({
...
         })}
       </select>
-      <IconCaretDown size={16} className="dropdown__icon-caret-down" />
+      <Icon
+        name={ICON_NAMES.ARROW_DOWN}
+        className="dropdown__icon-caret-down"
+      />
     </div>
   );
 };
```
Comment (georgewrmarshall) [inline ui/components/ui/dropdown/dropdown.js:50]: Just adding some sizing here

Screenshot 2023-02-08 at 6 16 00 PM

**Comment by darkwing** [comment]
Sorry @georgewrmarshall, this was meant to be a draft!

File: ui/components/ui/popover/popover.component.js [hunk 5]
```diff
@@ -67,10 +69,11 @@ const Popover = ({
...
           <h2 title="popover">
             {onBack ? (
-              <button
-                className="fas fa-chevron-left popover-header__button"
-                title={t('back')}
+              <ButtonIcon
+                iconName={ICON_NAMES.ARROW_LEFT}
+                ariaLabel={t('back')}
                 onClick={onBack}
+                color={Color.iconDefault}
```
Comment (georgewrmarshall) [inline ui/components/ui/popover/popover.component.js:76]: Adding size here to align with design system component but it feels very small cc @garrettbear @Akatori-Design. Request to add Size.MD to ButtonIcon

 
Screenshot 2023-02-08 at 6 49 12 PM

File: ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js (hunk 3 above)
Comment (darkwing) [reply ui/components/app/transaction-activity-log/transaction-activity-log-icon/transaction-activity-log-icon.component.js]: I updated the size here; looking at the code, do any of the icons look completely wrong?

File: ui/components/app/asset-list-item/asset-list-item.stories.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    tokenAddress: '0x2170ed0880ac9a755fd29b2688956bd959f933f8',
+    tokenSymbol: 'ETH',
+    tokenImage: './images/eth_logo.svg',
+    identiconBorder: true,
+  },
+};
+
+export const DefaultStory = (args) => <AssetListItem {...args} />;
+
+DefaultStory.storyName = 'Default';
```
Comment (georgewrmarshall) [inline ui/components/app/asset-list-item/asset-list-item.stories.js:54]: Adds a missing story. Used it for visual review. Can't hurt to include it for future visual review of this component

**Review by georgewrmarshall — DISMISSED** [review]
LGTM!

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Review by NidhiKJha — APPROVED** [review]

**Pull request merged**